Topic 1/src/point6/AbstractPayment.java

```diff
@@ -0,0 +1,46 @@
+package point6;
+
+public abstract class AbstractPayment {
```

Comment:
[High] It is not necessary to create an *Abstract* class for payment. You can use an *interface*